onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.h

```diff
@@ -83,7 +83,7 @@ namespace Dml
         std::vector<std::unique_ptr<onnxruntime::ComputeCapability>>
         GetCapability(
             const onnxruntime::GraphViewer& graph,
-            const std::vector<const onnxruntime::KernelRegistry*>& registries
+            const onnxruntime::IExecutionProvider::IKernelLookup& kernel_lookup
```

Comment:
@fdwr who on the DML side should check nothing is being overlooked here?

Reply:
@skottmckay Well @jeffbloo knows this function best in the DML EP.